src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIncrementalismTest.cs

```diff
@@ -28,7 +28,7 @@ public async Task BuildIncremental_SimpleMvc_PersistsTargetInputFile()
             var thumbprintLookup = new Dictionary<string, FileThumbPrint>();
 
             // Act 1
-            var result = await DotnetMSBuild("Build");
+            var result = await DotnetMSBuild("Build", runRestoreBeforeBuildOrPublish: false);
```

Comment:
Do not restore here since the test requires that none of the files change.

Reply:
I think @pranavkm needed this to happen to fix test flakiness. @pranavkm, can you please review to make sure this wouldn't cause any flakiness?